docs/static/docker.asciidoc

```diff
@@ -1,14 +1,23 @@
 [[docker]]
 === Running Logstash on Docker
 Docker images for Logstash are available from the Elastic Docker
-registry.
+registry. The base image is https://hub.docker.com/_/centos/[centos:7].
 
-The base image is https://hub.docker.com/_/centos/[centos:7] and the source
-code can be found on
-https://github.com/elastic/logstash-docker/tree/{branch}[GitHub].
+A list of all published Docker images and tags can be found at
+https://www.docker.elastic.co[www.docker.elastic.co]. The source code can be
+found on https://github.com/elastic/logstash-docker/tree/{branch}[GitHub].
 
-The images are shipped with https://www.elastic.co/products/x-pack[X-Pack]
-installed.
+==== Image types
+
+The images are available in two different configurations or "flavors". The
+`x-pack` flavor, which is the default, ships with X-Pack features
+pre-installed. The `oss` flavor does not include X-Pack, and contains only
+open-source Logstash.
```

Comment:
Super nitpicky, but the internal style guide says not to hyphenate "open source."

Reply:
Thanks. Super nitpicky is what we do!